src/vmm/src/builder.rs

```diff
@@ -1088,6 +1088,15 @@ pub mod tests {
         .unwrap()
     }
 
+    fn cmdline_contains(cmdline: &Cmdline, slug: &str) -> bool {
+        cmdline
+            .as_cstring()
+            .unwrap()
+            .into_string()
+            .unwrap()
+            .contains(slug)
+    }
+
```

Comment:
LGTM. Would there be value in adding this as a method in `Cmdline`?

Reply:
From my point of view there is no real value in adding this in `Cmdline` on rust-vmm side as long as there are only a couple of places where we use this. And in each of this places, we use this just for unit testing purposes.

Reply:
Closing this for unblocking merge of the PR; the discussion about rust-vmm upstreaming of this method will continue on the main thread, however.